h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -383,6 +387,18 @@ private RowData reconstructedToRowData(double[] reconstructed) {
     return result;
   }
 
+  private double calculateReconstructionError(double [] original, double [] reconstructed, double [] normmul, int numNumeric){
+    double l2 = 0;
+    int numIndex = 0;
+    double norm = 0;
+    for (int i = 0; i < numNumeric; i++) {
+      numIndex = i + (original.length - numNumeric);
+      norm = (normmul != null) ? normmul[i] : 1;
+      l2 += Math.pow((reconstructed[numIndex] - original[numIndex]) * norm, 2);
+    }
+    return  l2 / numNumeric;
```

Comment:
What if there are no numeric features (numNumeric == 0)?

Reply:
OMG, of course...